menas/ui/components/dataset/datasetDetail.controller.js

```diff
@@ -43,7 +43,15 @@ sap.ui.define([
       let cont = new ConformanceRuleDialog(this);
       let view = this.getView();
 
+      // Monitoring
+
+      this.MAX_RUNS = 500;
```

Comment:
Please look into using properties for this `MAX_RUNS` and a limit on line 121 in `/menas/src/main/scala/za/co/absa/enceladus/menas/repositories/MonitoringMongoRepository.scala`

Reply:
I propose we make this a separate issue

Reply:
Sure, made this a config property for now.

Reply:
That is actually a full solution. We may be don't need to create an issue then

Reply:
The property only works for the backend yet. UI still has a separate one